### PR TITLE
Display all TeamMembers of TeamAwesome [static]

### PR DESCRIPTION
**Pull request by jellebouwman**
Implemented a new page, team-members, with corresponding container- and team-member-overview components.

Added the appropriate styling.

### CODE REVIEW

File: components/team-member/TeamMember.jsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+          </div>
+          <div className="team-member-expand__starting-date">
+            <p className="team-member-expand__title">
+              February 2018
+            </p>
+            <p className="team-member-expand__subtitle">
+              Starting date
+            </p>
+          </div>
+        </div> */
```
Comment (RikFrieling) [inline components/team-member/TeamMember.jsx]: When creating a pull request you should remove the commented out code.

File: components/team-member/team-member.scss [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+  @include tablet {
+    width: 100%;
+  }
+}
+
+.team-member {
+  // border: solid 1px $color-accent-3;
+  // border-left: solid 4px $color-primary;
+  // border-radius: 4px;
```
Comment (RikFrieling) [inline components/team-member/team-member.scss]: Remove commented code

File: components/team-member/TeamMember.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            type="button"
+          >
+            <img
+              alt="arrow down sign"
+              className="team-member__expand-button-icon"
+              src="/static/icons/arrow-down-2.svg"
+            />
+          </button>
+        </div>
+        <div className={`
```
Comment (boris-arkenaar) [inline components/team-member/TeamMember.jsx]: Different name `team-member-expand` => different component.

File: pages/index.scss [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
-  @include container;
   padding-top: 70px;
 
   @include tablet {
     padding: 70px 0;
-
   }
 }
+
+.team-member__page-wrapper {
```
Comment (boris-arkenaar) [inline pages/index.scss]: Why not `team-member-page`? Same would go for `timesheet-page`. Naming is not consistent at the moment.

**Pull request merged**